gatsby_fp/src/components/fieldComponents/generalSection.js

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export default function Section({ fieldLabel, fieldData }) {
```

Comment:
We should probably change the name here to GeneralSection too, for clarity.

Reply:
I only named the file "generalSection" as a rename (actually for the same reason that I am about to send the commit with all the filenames being NameX) , so I am just going to revert it back to its original name "section.js" -- (now Section.js) with PascalCase

Reply:
renamed to GeneralSection